projects/tinkerbell/tinkerbell-chart/chart/templates/boots/deployment.yaml

```diff
@@ -27,6 +27,8 @@ spec:
           {{- end }}
           {{- if .Values.boots.env }}
           env:
+            - name: "BOOTS_KUBE_NAMESPACE"
```

Comment:
is there a change in eks-a required to set this new value?

Reply:
No change required. It uses the already set namespace (in eks-a) to populate it.

Reply:
what is this value used for?
Will this always be same as boots namespace under metadata? If so, can we get the namespace using something like this instead? https://stackoverflow.com/a/46046153

Reply:
this version of Boots needs a namespace. This is the commit that introduced this. https://github.com/tinkerbell/boots/commit/b9b0f72614c976e6623e76ef16a32d44b2a45b9b

Is this, https://stackoverflow.com/a/46046153, something you are wanting implemented with Helm?

Reply:
no I was thinking we could get the namespace from that file instead of using an env but I just realized that might not work because boots runs outside of kubernetes for the initial bootstrap